Description: some cosmetics backported
 - display volume percent 
 - fix worldclock size

Author: Alf Gaida <agaida@siduction.org>
Last-Update: 2017-11-05

--- lxqt-panel-0.12.0.orig/plugin-volume/lxqtvolume.cpp
+++ lxqt-panel-0.12.0/plugin-volume/lxqtvolume.cpp
@@ -286,7 +286,7 @@ void LXQtVolume::showNotification(bool f
             if (m_defaultSink->mute())
                 m_notification->setSummary(tr("Volume: muted"));
             else
-                m_notification->setSummary(tr("Volume: %1").arg(QString::number(m_defaultSink->volume())));
+                m_notification->setSummary(tr("Volume: %1%").arg(QString::number(m_defaultSink->volume())));
             m_notification->update();
         }
     }
--- lxqt-panel-0.12.0.orig/plugin-worldclock/lxqtworldclock.cpp
+++ lxqt-panel-0.12.0/plugin-worldclock/lxqtworldclock.cpp
@@ -132,7 +132,10 @@ void LXQtWorldClock::updateTimeText()
 
     if (!isUpToDate)
     {
+        const QSize old_size = mContent->sizeHint();
         mContent->setText(tzNow.toString(preformat(mFormat, timeZone, tzNow)));
+        if (old_size != mContent->sizeHint())
+            mRotatedWidget->adjustContentSize();
         mRotatedWidget->update();
         updatePopupContent();
     }
